mobile/src/bundles/employer/components/candidate-card/candidate-card.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+import {
+    Badge,
+    Button,
+    Tag,
+    Text,
+    View,
+} from '~/bundles/common/components/components';
+import { BadgeSize, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+import { type ValueOf } from '~/bundles/common/types/types';
+import { type BsaBadgeStepBadgesTitle } from '~/bundles/talent/enums/enums';
+
+import { styles } from './styles';
+
+type BadgeName = ValueOf<typeof BsaBadgeStepBadgesTitle>;
+
+type Properties = {
+    id: number;
+    salaryExpectation: number;
+    jobTitle: string;
+    location: string;
+    experienceYears: number;
+    englishLevel: string;
+    description: string;
+    published: string;
+    hardSkills: string[];
+    badges: {
+        label: BadgeName;
+        value: string | number;
+    }[];
+};
```

Comment:
Pick types from user data types 
Pick<UserData, 'salaryExpectation' |  'jobTitle ' etc>

Reply:
and pass it just as user data why separate prop for each filed

Reply:
I created type folder for storing type of Candidate because we need to use it in two places, and also I used `UserDetailsCreateRequestDto` from shared folder since we do not have type UserData for now.